data-esnext.js

```diff
@@ -2676,7 +2676,8 @@ exports.tests = [
       res: {
         babel: babel.corejs,
         typescript: typescript.corejs,
-        firefox52: false,
+        firefox2: false,
```

Comment:
Did you explicitly test Firefox 2? If not, it shouldn’t be in here.

Either way, please keep the explicit v52 results.

Reply:
Yes, I have tested firefox 2 and every ESR version up to 52, and also versions 53..58.  I have also tested with many versions of Chrome and Edge.  I don't see the point of adding every version.

Reply:
Generally, these entries should represent explicit human test results, and thus should never be removed (unless incorrect).

I agree having all of those versions in the list is impractical; maybe we should settle for including 2, 57, and 58?

Reply:
Done.